CreeDictionary/CreeDictionary/views.py

```diff
@@ -107,7 +111,7 @@ def search_results(request, query_string: str):  # pragma: no cover
     results = Wordform.search_with_affixes(query_string)
     return render(
         request,
-        "CreeDictionary/word-entries.html",
+        "CreeDictionary/search-results.html",
```

Comment:
Are there any tests that use to reference /word-entries that now need to look for /search-results? 
If not, should there be tests for the new /search-results path?

Reply:
This a rename (refactor) to a _template file_. It's not a URL path! The URLs have all stayed the same. That said, I'd be interested in hearing how we can strengthen our tests!

Reply:
Here is the renamed file in question: https://github.com/UAlbertaALTLab/cree-intelligent-dictionary/pull/616/files#diff-58ca8956f4bfbe13e67b376ee3b4e5d1ab1445330bcda3eb43c2125bd57b1f80

Reply:
Ah I see. We should go over the tests in a different PR if they aren't directly related to these changes, then.